### PR TITLE
added new images for the home and intro pages

### PR DESCRIPTION
**Pull request by iamfortune**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -89,6 +89,12 @@ const config = {
             position: 'left',
             label: 'Guides',
           },
+          // {
+          //   type: "docSidebar",
+          //   position: "left",
+          //   sidebarId: "learn",
+          //   label: "Learn",
+          // },
```
Comment (mcollina) [inline docusaurus.config.js:97]: please remove, dont comment out

Comment (iamfortune) [reply docusaurus.config.js]: Removed, needed this for the new section of the docs that is currently WIP

**Review by mcollina — CHANGES_REQUESTED** [review]
The dark mode image has not been update, it still reports the Payment service as within Platformatic Runtime

**Comment by iamfortune** [comment]
> The dark mode image has not been update, it still reports the Payment service as within Platformatic Runtime

There is no need for this, will make a commit to remove the dark mode images as we do not use it in the docs presently.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**